src/assets/variables.scss

```diff
@@ -20,11 +20,14 @@
  *
  */
 
-//messages list max width
-$messages-list-max-width: 800px;
-
-//message utils width
-$message-utils-width: 100px;
+/** Messages list dimensions:
+ * - text max width: 70em (recommended by W3C standard)
+ * - utils-width: 52px (group avatar with paddings) + 120px (date and checkmark) + 4*4px (date paddings)
+ * - list max width: text width + utils width + 4 + 8 (paddings)
+ */
```

Comment:
Can you please add totals in px here ? so not to recalculate this every time we check it later :p

Reply:
Done